copter/source/docs/high-loop-rate-tuning.rst

```diff
@@ -0,0 +1,41 @@
+.. _high-loop-rate-tuning:
+
+===========================
+Aggressive Rate Loop Tuning
+===========================
+
+Normally, the attitude rate PID controllers update at the :ref:`SCHED_LOOP_RATE<SCHED_LOOP_RATE>`, the main loop speed. This means that PID rate error corrections can happen only at 1/loop rate. For 400hz, this is 2.5ms. In ArduCopter 4.7 and later, it is possible to run the PID attitude rates closer to the rate at which the gyros provide measurements, which can be up to 4KHz, resulting in 2.5uS rate corrections. Tuning is more sensitive and complex, but for aerobatic applications on vehicles with large thrust to inertia ratios (ie very agile) could yield much more precise and responsive control.
```

Comment:
It's also because the filtering can only be done at loop rate/2

Reply:
will add